my-app/src/components/Cocina.jsx

```diff
@@ -0,0 +1,179 @@
+import React, { useEffect, useState, Fragment } from "react";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
+
+import Meseros from "./Meseros";
+import db from "../firebase";
+import Fecha from "./Fecha";
+
+const Cocina = () => {
+  // TODO: Cambiar de nombre a  "orders", "setOrders"
```

Comment:
si es algo clave déjalo pero si no te sirve quítalo
```suggestion
  // TODO: Cambiar de nombre a  "orders", "setOrders"
```